pom.xml

```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <artifactId>mod-inventory</artifactId>
   <groupId>org.folio</groupId>
-  <version>12.1.0-SNAPSHOT</version>
+  <version>12.2.0-SNAPSHOT</version>
```

Comment:
Why has this version been changed?

Version 12.1.0 has not been released yet, and these changes are minor, meaning they fit within the bounds of the current 12.0.0 -> 12.1.0 increment.

Is this because of the conversation we had about changing the major implementation version in the other pull request?

Reply:
From the https://dev.folio.org/guidelines/contributing/#version-numbers

> For modules the middle part should be incremented if you implemented an addition to the API(s), (the minor version of the particular API has been changed). For non-modules it may also mean any additional functionality. For both, the change must be backwards compatible with respect to any client code or agents.

according to this " (the minor version of the particular API has been changed)" I decided to change the version of impl.

Reply:
> From the https://dev.folio.org/guidelines/contributing/#version-numbers
> 
> > For modules the middle part should be incremented if you implemented an addition to the API(s), (the minor version of the particular API has been changed). For non-modules it may also mean any additional functionality. For both, the change must be backwards compatible with respect to any client code or agents.
> 
> according to this " (the minor version of the particular API has been changed)" I decided to change the version of impl.

Alas, that guidance is incorrect in the context of pull requests (and probably out of date in other ways), the minor version number should have already been incremented during the last formal release.

Otherwise pretty much every pull request would require the implementation version to change and that would just unnecessarily waste version numbers.

Please reset the implementation version number. Apologies for the confusion.

Does that make sense?

Reply:
It's clear. Thanks